### ISSUE TITLE
[3.4] and [4.0] Blank parts of the error/warning text box in the script pane are not text-selectable.

### ISSUE THREAD

**Issue by Proggle** [godotengine/godot#57781]
### Godot version
v3.4.stable.official [206ba70f4]
v4.0.alpha1.official [31a7ddbf8]

### System information

Windows 10

### Issue description

When you have warnings or errors like this, if you want to select part of the text box, it doesn't work as expected

![image](https://user-images.githubusercontent.com/89282845/152905247-28f0ed5a-d21f-4417-a763-e8c1402fd5cc.png)

While you can click and drag to select text if you put your cursor exactly onto a letter, if you put it in the blank space to the right, the cursor will turn into the little text selecting ][ shape, but won't actually select any text

![image](https://user-images.githubusercontent.com/89282845/152905501-bfbb5266-3a83-4488-bef7-d266be78c04b.png)

What it should do is start selecting on the same line, from the right-most character, just like a normal text editor.

### Steps to reproduce

Get an error or warning, and try to select the text.

**Comment by Calinou** [comment]
I believe this is an issue with RichTextLabel in general, rather than an editor issue.

**Comment by Proggle** [comment]
> I believe this is an issue with RichTextLabel in general, rather than an editor issue.
> 
> @Proggle Can you reproduce this on 3.4.2?
> 
> cc @bruvzg

I've got it in 3.4 stable at least.  Seems like anything after the end of a line has this selection issue.  I used this function to throw a bunch of warnings
```
func foo(alpha,beta,delta,gamma,a,b,c,d,e,f,g):
	pass
```

And the areas I've boxed are unselectable as well.

![image](https://user-images.githubusercontent.com/89282845/153050299-01a7599c-f338-4d35-ada0-905f1210de28.png)

The output pane has something similar, but the non-working area is only the part to the right and below the last line.

https://user-images.githubusercontent.com/89282845/153050980-aff4a790-673a-4618-b92f-f4e6716513ff.mp4
